common/djangoapps/edeos/migrations/0002_auto_20190723_0226.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('edeos', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='usersociallink',
+            name='platform',
+            field=models.CharField(max_length=50, choices=[(b'facebook', b'Facebook')]),
```

Comment:
Looks like we missed this migration before :( 
here: https://github.com/raccoongang/edx-platform/commit/fb96b6537727fabf0ca56671ecb514b6d5811c97

Reply:
I add the migration

Reply:
ok I will add this Migration in this tasks.

Reply:
ok